src/main/java/Entities/userFactory.java

```diff
@@ -1,4 +1,5 @@
 package Entities;
 
 public interface userFactory {
+    User createUser(String artistName, String username, String password);
```

Comment:
Re @rafaelgacesa's comment about interface segregation principle: technically, it's fine because that specifies reliance on a _method_ that is not used (to create a regular user, I assume the userFactory will still be called; it's the parameter artistName that is unused). 

Instead, I would recommend overloading the method createUser in the interface -- i.e. have User createUser(artistName, username, password) as you do now and add another line underneath with User createUser(username, password) so that when you're creating a regular user, you don't need to pass anything in for the artist name at all.

Reply:
Wouldn’t overloading it then create a violation of the interface segregation principle ?

Reply:
Hmm... maybe you're right. I didn't think you'd have to implement both versions when implementing the interface, but now that I think about it, you probably do. In that case, what Jessica currently has should be fine?

Reply:
@chrstinalin  do you mean my initial comment or raf's suggestion? I realize now that my initial suggestion wouldn't really help lol.

Reply:
I was replying to your original comment since that's the only thing that was coming up for me, lol. I agree with what he's saying.

Reply:
I went to the office hour and the ta suggested to just have the userFactory for creating both user object, what do you guys think?

Reply:
One user factory with two methods?